extensions/hibernate-search-standalone-elasticsearch/runtime/src/main/java/io/quarkus/hibernate/search/standalone/elasticsearch/runtime/management/HibernateSearchStandaloneManagementConfig.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.hibernate.search.standalone.elasticsearch.runtime.management;
+
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+@ConfigMapping(prefix = "quarkus.hibernate-search-standalone.management")
+@ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public interface HibernateSearchStandaloneManagementConfig {
+
+    /**
+     * Root path for reindexing endpoints.
+     * This value will be resolved as a path relative to `${quarkus.management.root-path}`.
+     *
+     * @asciidoclet
+     */
+    @WithDefault("hibernate-search/")
```

Comment:
There won't be a conflict between the two extensions here?

Reply:
There will be, thanks for spotting this.

I'll change this to `hibernate-search/standalone/`.